build/yaml/botbuilder-dotnet-ci.yml

```diff
@@ -44,15 +44,32 @@ variables:
 stages:
 - stage: Build
   jobs:
-  - job: Debug_Windows_Configuration
+  - job: Debug_Windows_Configuration_21
     variables:
       BuildConfiguration: Debug-Windows
+      BuildTarget: 'netcoreapp21' # set the TargetFramework property for tests to use netcoreapp2.1
     steps:
     - template: ci-build-steps.yml
     - template: ci-test-steps.yml
-  - job: Release_Windows_Configuration
+  - job: Debug_Windows_Configuration_31
+    variables:
+      BuildConfiguration: Debug-Windows
+      BuildTarget: 'netcoreapp31' # set the TargetFramework property for tests to use netcoreapp3.1
+    steps:
+    - template: ci-build-steps.yml
+    - template: ci-test-steps.yml
+  - job: Release_Windows_Configuration_21
+    variables:
+      BuildConfiguration: Release-Windows
+      BuildTarget: 'netcoreapp21' # set the TargetFramework property for tests to use netcoreapp2.1
+      NoCoverageUpload: true
+    steps:
+    - template: ci-build-steps.yml
+    - template: ci-test-steps.yml
+  - job: Release_Windows_Configuration_31
     variables:
       BuildConfiguration: Release-Windows
+      BuildTarget: 'netcoreapp31' #set the TargetFramework property for tests to use netcoreapp3.1
```

Comment:
> #set the TargetFramework property for tests to use netcoreapp3.1.

We should add a space between `#` and `set` comment, maybe on `necoreapp 3.1` too.

Reply:
yes, I missed the first space there.
The netcoreapp3.1 name is how the name is set inside the properties, for example `<TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>`